frameworks/hdfs/universe/config.json

```diff
@@ -63,55 +63,51 @@
           "enum": ["OFF", "FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE", "ALL"],
           "default": "INFO"
         },
-        "tls": {
+        "security": {
+          "description": "HDFS security settings.",
           "type": "object",
-          "description": "TLS configuration properties.",
           "properties": {
-            "enabled": {
-              "description": "Enable TLS support (requires Strict mode).",
-              "type": "boolean",
-              "default": false
-            }
-          }
-        },
-        "kerberos": {
-          "type": "object",
-          "description": "Kerberos configuration properties.",
-          "properties": {
-            "enabled": {
-              "description": "Enable kerberos authentication.",
-              "type": "boolean",
-              "default": false
-            },
-            "kdc_host_name": {
-              "type": "string",
-              "description": "The name or address of a host running a KDC for the realm.",
-              "default": "kdc.marathon.autoip.dcos.thisdcos.directory"
-            },
-            "kdc_host_port": {
-              "type": "string",
-              "description": "The port of the host running a KDC for that realm.",
-              "default": "2500"
-            },
-            "keytab_secret": {
-              "type": "string",
-              "description": "A DC/OS Secret Store path.  The contents of this secret should be a keytab containing the credentials for all HDFS servers.",
-              "default": "__dcos_base64__hdfs_keytab"
+            "tls": {
+              "type": "object",
+              "description": "TLS configuration properties.",
+              "properties": {
+                "enabled": {
+                  "description": "Enable TLS support (requires Strict mode).",
```

Comment:
Note that this only requires enterprise, and not strict. I was looking at INFINITY-2583 and noted that Cassandra and Elastic have the "correct" description, but Kafka doesn't.